tools/cldr-code/src/main/java/org/unicode/cldr/test/DisplayAndInputProcessor.java

```diff
@@ -197,8 +197,8 @@ public class DisplayAndInputProcessor {
     public static final Set<String> LANGUAGES_USING_MODIFIER_APOSTROPHE =
```

Comment:
😱 don't like these kinds of lists… in code…

Reply:
Agreed. We should drive this off of whether the language's exemplars include modifier apostrophe.